### ISSUE TITLE
Add support Tuya IR Remote

### ISSUE THREAD

**Issue by sgilyin** [rospogrigio/localtuya#515]
Hello!
Let's add support Tuya IR Remote?

**Comment by tcass1977** [comment]
I agree.  Just need a bump in the right direction for intel gathering.  I/ve been reading up on the broadlink integration and messing around with the ir bridge mockup on the IoT platform with tuya developer.  Something, should be attainable.

**Comment by KirkKirk** [comment]
In the same boat. 
I bought Tuya Smart WIFI IR Universal Infrared Wireless Remote Controller and I can't integrate it into Home Assistant.
It works in Tuya Smart perfectly and it is voice-controlled in Google Assistant.
Will be awesome if LocalTuya can integrate all Tuya supported devices.

**Comment by jyhur805** [comment]
Plz.

**Comment by lisankai93** [comment]
please~~~

**Comment by BjornRonnestad** [comment]
Yes please👍

**Comment by casper255** [comment]
Please

**Comment by rdaguerre** [comment]
same problem. same wish

**Comment by mytasker** [comment]
please!
I got this device:
https://www.aliexpress.com/item/1005003177136123.html?spm=a2g0o.9042311.0.0.688b4c4durkPA9

And I could integrate Humidity and Temperature with LocalTuya, but not the IR.

Please, help us.

**Comment by xlucasfelix** [comment]
I would also really like this to be added. I bought two Smart IR Tuya in hopes they would work in Home Assistant, but they are not compatible (they show up in HA but don't provide any entities)

**Comment by johnsonchi3689** [comment]
same problem. same wish

**Comment by deadxd** [comment]
same problem. same wish <3

**Comment by bepstein111** [comment]
+1

**Comment by Kahera** [comment]
+1

**Comment by yarafie** [comment]
+1

**Comment by broyuken** [comment]
Same here. Is there anything we can provide to help get this added?

**Comment by IdefixRC** [comment]
+1
would be awesome if support for these IR/RF Blasters can be added. They work great via the Tuya App (or integration via Scenes) but not directly. Thanks !
Product Link: https://www.aliexpress.com/item/1005005940671116.html

**Comment by vadym-osipchuk** [comment]
Same here with that IR/RF Blaster
